apps/api/src/domain/models/term/commands/create-term/term-created.event-handler.integration.spec.ts

```diff
@@ -63,7 +71,28 @@ describe(`TermCreatedEventHandler`, () => {
 
     beforeAll(async () => {
         const moduleRef = await Test.createTestingModule({
-            providers: [CommandInfoService, TermCreatedEventHandler],
+            providers: [
+                CommandInfoService,
+                TermCreatedEventHandler,
+                {
+                    provide: 'TOKENIZER_PROVIDER',
```

Comment:
This provider can't be a shared import with the term module because this is a test?

Reply:
We don't want it in the term module. We are going to tokenize text on every categorizable (resource or note). Also, we want to abstract the internals from the resource modules. Eventually, we will use `spaCy` to do tokenization, but integrating python in this way seemed premature. It's likely that this happens via the messaging queue for async (truly out of process) event consumers, which will be able to be written in any language. But I think the point is a good one, so I'll make an `NLP` module for now.